include/context.h

```diff
@@ -87,7 +87,8 @@ void r_context_begin_step(const gchar *name, const gchar *description,
  * @param description that is emitted via DBus on begin/end.
  *   A printf-like format string.
  */
-void r_context_begin_step_formatted(const gchar *name, gint substeps, const gchar *description, ...);
+void r_context_begin_step_formatted(const gchar *name, gint substeps, const gchar *description, ...)
+	__attribute__((__format__(__printf__, 3, 4)));
```

Comment:
Seems like uncrustify doesn't like it to be placed this way. Thus we either need to simply append it to the end of the declaration line or maybe find the right bit in uncrustify to accept this...

Reply:
OK, I've simply accepted uncrustify's suggestion.